### DISCUSSION TITLE
Threads not working properly in my INBOX after upgrading 2.27.3 -> 2.32.0

### DISCUSSION THREAD

**Discussion by pepelton** [the-djmaze/snappymail#1399] [Q&A]
After the upgrade in my INBOX threads do not work properly:

In the message list the threads are shown correctly but when I click to enter the thread I do not get a list of messages in the thread instead Snappy shows me again the whole INBOX message list without threads. 

The strange thing is that the threads work properly in my Sent folder. Also if I switch to another mailbox I have configured to Snappy, the INBOX threads work correctly. 

Is there some cache or something that I could try clearing to see if it helps?

**Comment by the-djmaze** [comment]
Best is to enable debug and log modes.
Then more info is gathered to find the problem

**Comment by simonbuehler** [comment]
Subscribe to this - I have the same issue here

**Comment by the-djmaze** [comment]
Go to `?admin#/domains` and open the domain config.

Is `Limits` => `Messages` set to `0`?

**Comment by simonbuehler** [comment]
they were both 10000 and 0 in two accounts but same behavior
